.gitignore

```diff
@@ -1 +1,2 @@
 node_modules
+serve_pr/[0-9]*/
```

Comment:
this is for running the script directly right?

Reply:
Yes, exactly.